ipxe/files.go

```diff
@@ -0,0 +1,11 @@
+package ipxe
+
+import (
+	"embed"
+)
```

Comment:
pretty sure golangci-lint will complain about this sooner or later in this repo (not gofumpted), should be `import "embed"`

Reply:
I've modified it to a single line.